DS/DS-Session1/Queue/QueueUsingArray.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * It is implementing Queue interface which has methods related to queue and it has
+ * implemented using array. Front is for removal and Rear is for insertion in Queue
+ * 
+ * @author Shubham Jain
+ *
+ */
+public class QueueUsingArray implements Queue {
```

Comment:
@meta-shubham-jain 
Queue using array may be more sensible if you user circular array

Reply:
Circular Queue is in second Assignment to implement